File: E-You App/enviromental-app-develop_dragos/Android/app/src/main/java/com/example/greenlight/di/MainActivityModule.java
package com.example.greenlight.di;

import com.example.greenlight.MainActivity;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class MainActivityModule {

    @ContributesAndroidInjector
    abstract MainActivity contributeMainActivityModule();

}

File: LICENTA_GREENLIGHT/Android/app/src/main/java/com/ciurezu/gheorghe/dragos/greenlight/di/MainActivityModule.kt
package com.ciurezu.gheorghe.dragos.greenlight.di

import com.ciurezu.gheorghe.dragos.greenlight.MainActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class MainActivityModule {
    @ContributesAndroidInjector
    abstract fun contributeMainActivity(): MainActivity
}